packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -233,7 +234,7 @@ export class VaAccordion {
             (this.accordionContainer = accordionContainer)
           }
         >
-          {!openSingle ? (
+          {(!openSingle && !isInsideSearchFilter) ? (
```

Comment:
The designs do not want the expand/collapse all toggle on the va-search-filter component so this condition will handle removing it.

![Screenshot 2025-04-17 at 3 29 34 PM](https://github.com/user-attachments/assets/434e6346-a6b7-4386-8ab2-e5fb590ad4b7)